detox/test/e2e/06.device.test.js

```diff
@@ -39,6 +42,15 @@ describe('Device', () => {
     await expect(element(by.text('Hello!!!'))).toBeVisible();
   });
 
+  it('getViewHierarchyXml() - should return a valid UI hierarchy', async () => {
```

Comment:
I would suggest this approach:

```js
  describe('getViewHierarchyXml()', () => {
    beforeAll(async () => {
      await device.launchApp({newInstance: true});
      await element(by.text('Sanity')).tap();
      await element(by.text('Say Hello')).tap();
    });

    it(':ios: @rn71 should return a valid UI hierarchy', () =>
      jestExpect(device.getViewHierarchyXml()).resolves.toMatchSnapshot());

    it(':ios: @rn73 should return a valid UI hierarchy', () =>
      jestExpect(device.getViewHierarchyXml()).resolves.toMatchSnapshot());

    it(':android: @rn71 should return a valid UI hierarchy', () =>
      jestExpect(device.getViewHierarchyXml()).resolves.toMatchSnapshot());

    it(':android: @rn73 should return a valid UI hierarchy', () =>
      jestExpect(device.getViewHierarchyXml()).resolves.toMatchSnapshot());
  });
```

Reply:
I less agree with this approach because we will forget about in in rn74